public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This task isn't a PWA, so we don't need a manifest. Please remove this `manifest.json` file